blocks/address/turbo.json

```diff
@@ -1,24 +1,24 @@
 {
   "extends": ["//"],
   "pipeline": {
-    "schema": {
+    "codegen": {
       "inputs": ["./package.json"],
-      "outputs": ["./src/types.gen.ts"]
+      "outputs": ["./src/types/generated/**"]
     },
     "build": {
-      "dependsOn": ["^build", "schema", "format"],
+      "dependsOn": ["^build", "codegen", "format"],
       "outputs": ["./dist/**"]
     },
     "lint:tsc": {
-      "dependsOn": ["schema"]
+      "dependsOn": ["codegen"]
     },
```

Comment:
Not asking for a change, but we don't need this anymore as it's defined in the root. (Technically the root also says `"^build"` which might be important if we ever change how the eslint or tsconfig stuff is 'built'

Reply:
Did wonder about these. Will leave for now and we can amend when we next touch these